### PR TITLE
Developer-friendly repo structure

### PR DESCRIPTION
**Pull request by VladislavFitz**

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+- Select `Settings` -> `Configuration`
+- Click **New application setting**
+- Set name to `PORT` and value to `80`, click **OK** and then **Save**
+- You're all set. Go to the `Overview` section. You can find the URL of your application which can be used as the endpoint in the index settings.
+
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
```
Comment (Haroenv) [inline docs/setup.md:257]: they were already there, but what is the br for?

**Review by Haroenv — APPROVED** [review]
makes more sense like this indeed

File: docs/setup.md (hunk 1 above)
Comment (bodinsamuel) [reply docs/setup.md:257]: same comment

**Review by bodinsamuel — APPROVED** [review]
Feels more correct 🙌🏻

File: docs/setup.md (hunk 1 above)
Comment (VladislavFitz) [reply docs/setup.md:257]: This brs are to layout properly the images aligned to right (you can try to remove them in the editing mode and see the preview). If you know a more proper way to achieve this, don't hesitate to commit.

**Pull request merged**